src/App.tsx

```diff
@@ -244,22 +253,38 @@ const getL1ToL2MessagesAndDepositMessages = async (
         );
     }
     const l2Provider = new StaticJsonRpcProvider(l2RpcURL);
-    const l1ToL2MessagesWithNetwork: L1ToL2MessageReaderWithNetwork[] = (
-      await l1TxnReceipt.getL1ToL2Messages(l2Provider)
-    ).map((l1ToL2Message) => {
-      return Object.assign(l1ToL2Message, { l2Network });
-    });
+    const isClassic = await l1TxnReceipt.isClassic(l2Provider);
```

Comment:
we shouldn't ever be seeing classic l1 to l2 messages anymore, so it seems fine to disregard IMO (also fine to leave as is)